esbuild.config.js

```diff
@@ -0,0 +1,51 @@
+const esbuild = require('esbuild');
+
+const { NodeModulesPolyfillPlugin } = require("@esbuild-plugins/node-modules-polyfill");
+const { NodeGlobalsPolyfillPlugin } = require("@esbuild-plugins/node-globals-polyfill");
+
+const path = require('path');
+const fs = require('fs');
+
+
+function copydynload() {
+  const srcDir = path.resolve('./src/dynload');
+  const destDir = path.resolve('./lib/dynload');
+
+  if (fs.existsSync(destDir)) {
+    fs.rmSync(destDir, { recursive: true });
+  }
+  fs.mkdirSync(destDir, { recursive: true });
+
+  fs.readdirSync(srcDir).forEach(file => {
+    fs.copyFileSync(path.join(srcDir, file), path.join(destDir, file));
+  });
+}
```

Comment:
Why do we need custom logic for dynload files? Should it be exported as part of the main mambajs library?